projects/laji/src/app/shared-modules/document-viewer/document-annotation/document-annotation.component.ts

```diff
@@ -330,16 +329,13 @@ export class DocumentAnnotationComponent implements AfterViewInit, OnChanges, On
         this.hasEditors = true;
       }
 
-      if (this.document.gatherings) {
-        this.document.gatherings.forEach(gathering => {
-          if (gathering.units) {
-            const i = gathering.units.find(unit => unit.unitId === this.highlight);
-            if (i) {
-              return this.unitExist = true;
-            }
-          }
-        });
-      }
+      this.unitOrImgExists = this.document.gatherings?.some(({units}) =>
+        (units || []).some(unit =>
+          this.identifying
```

Comment:
`this.identifying` is not needed here either. Same kind of solution applies to this that what was already implemented to ImagesComponent. (I'm not sure how the identifying input should be thought in the context of annotation viewer and that's why I'd try to remove that (or possibly rename it to something more descriptive in this context))